installation/universal/01-setup-ingress-controller.sh

```diff
@@ -6,14 +6,16 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd "${SCRIPT_DIR}"
 source "_settings.sh"
 
+kubectl create --dry-run=client -o yaml namespace ingress-nginx | kubectl apply -f
+
+kubectl apply -n ingress-nginx -f ./nginx-ingress/configmap.yaml
```

Comment:
Would it make sense to rename the `nginx-ingress` directory to `ingress-nginx`? This way it matches the chart (and namespace) we are using?

Reply:
Yes, makes sense!